app/models/provider_relationship_permissions.rb

```diff
@@ -36,6 +36,19 @@ def permit?(permission, provider)
     end
   end
 
+  scope :governs_open_course, lambda {
+    course_joins_sql = <<-SQL
+      JOIN courses
+      ON provider_relationship_permissions.training_provider_id = courses.provider_id
+      AND provider_relationship_permissions.ratifying_provider_id = courses.accredited_provider_id
+    SQL
+    joins(course_joins_sql).merge(Course.current_cycle.open_on_apply).distinct
+  }
+
+  def governs_open_course?
```

Comment:
We went with this so that things like:
```
ProviderRelationshipPermissions.find(1).governs_open_course?
```
and also
```
ProviderRelationshipPermissions.governs_open_course
```
read like sentences.

It would also work as `has_open_course`, but rubocop complained about that. `with_open_course?` doesn't read as well though.

I think we can just ignore rubocop on this one